apps/allocations/app/components/Card/Budget.js

```diff
@@ -0,0 +1,213 @@
+  screenSize,
+}) => {
+  const theme = useTheme()
+  const newAllocation = () => {
+    onNewAllocation(id, name, amount)
+  }
+  const edit = () => {
+    onEdit(id)
+  }
+  const deactivate = () => {
+    onDeactivate(id)
+  }
+
+  const reactivate = () => {
+    onReactivate(id)
+  }
+
+  if (inactive) {
+    return (
+      <StyledCard screenSize={screenSize}>
+        <MenuContainer>
+          <ContextMenu>
+            <ContextMenuItem onClick={reactivate}>
+              <IconView />
+              <ActionLabel>Reactivate</ActionLabel>
+            </ContextMenuItem>
+          </ContextMenu>
+        </MenuContainer>
+        <CardTitle theme={theme}>{name}</CardTitle>
+        <StatsContainer>
+          <StyledStats>
+            <StatsValueBig theme={theme}>
+              <Text>Inactive</Text>
+            </StatsValueBig>
+          </StyledStats>
+        </StatsContainer>
+      </StyledCard>
+    )
+  }
+
+  return (
+    <StyledCard screenSize={screenSize}>
+      <MenuContainer>
+        <ContextMenu>
+          <ContextMenuItem onClick={newAllocation}>
+            <IconPlus />
+            <ActionLabel>New Allocation</ActionLabel>
+          </ContextMenuItem>
+          <ContextMenuItem onClick={edit}>
+            <IconEdit />
+            <ActionLabel>Edit</ActionLabel>
+          </ContextMenuItem>
+          <ContextMenuItem onClick={deactivate}>
+            <IconProhibited />
+            <ActionLabel>Deactivate</ActionLabel>
+          </ContextMenuItem>
+        </ContextMenu>
+      </MenuContainer>
+      <CardTitle theme={theme}>{name}</CardTitle>
+      <StatsContainer>
+        <StyledStats>
+          <StatsValueBig theme={theme}>
+            {displayCurrency(BigNumber(amount))}
+            <Text>{' ' + currency + ' per period'}</Text>
+          </StatsValueBig>
+          <StatsValueBig css={{ paddingTop: '24px' }} theme={theme}>
+            <ProgressBar
+              color={String(theme.accentEnd)}
+              value={BigNumber(allocated).div(amount).toNumber()}
+            />
+          </StatsValueBig>
+          <StatsValueSmall css={{
+            color: theme.content,
+            paddingTop: '8px',
+          }}>
+            {displayCurrency(BigNumber(amount).minus(allocated))}
+            <Text>{' ' + currency + ' below limit'}</Text>
+          </StatsValueSmall>
+          <StatsValueSmall css={{
+            color: theme.contentSecondary,
+            paddingTop: '4px',
+          }}>
+            {BigNumber(amount)
+              .minus(allocated)
+              .div(amount)
+              .multipliedBy(100)
+              .dp(0)
+              .toString()}
+            <Text>{'% remaining'}</Text>
+          </StatsValueSmall>
+        </StyledStats>
+      </StatsContainer>
+    </StyledCard>
+  )
+}
+
+Budget.propTypes = {
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  amount: PropTypes.string.isRequired,
+  currency: PropTypes.string.isRequired,
```

Comment:
These props will need some format changes, I will try to define them better, just a self-reminder for me